api/apps/services/dataset_api_service.py

```
@@ -1543,6 +1661,253 @@ async def has_any_wiki(dataset_id: str, tenant_id: str):
+    grouped: dict[str, dict] = {}
+    for row in rows.values():
+        tid = _row_template_id(row)
+        stamped_kind = (row.get("compilation_template_kind_kwd") or "").strip()
+        row_kind = stamped_kind or _template_meta(tid) or ""
+        if _resolve_dataset_structure_kind(row_kind) != resolved_kind:
+            continue
+
+        try:
+            graph = json.loads(row.get("content_with_weight") or "{}")
+        except Exception:
+            continue
```

Ruff (0.15.21)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)